agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/instrumentation/sdk/BytecodeUtilImpl.java

```diff
@@ -55,29 +55,32 @@
 // supporting all properties of event, metric, remove dependency and page view telemetry
 public class BytecodeUtilImpl implements BytecodeUtilDelegate {
 
-    private static final Tracer tracer = OpenTelemetry.getGlobalTracer("");
-
     private static final Logger logger = LoggerFactory.getLogger(BytecodeUtilImpl.class);
 
     private static final AtomicBoolean alreadyLoggedError = new AtomicBoolean();
 
     @Override
-    public void trackEvent(String name, Map<String, String> properties, Map<String, Double> metrics) {
+    public void trackEvent(String name, Map<String, String> properties, Map<String, String> tags, Map<String, Double> metrics,
+                           String instrumentationKey) {
```

Comment:
where are we actually using the instrumentationKey here?

Reply:
no, good point, I have removed 👍